src/runtime/pythonexception.cs

```diff
@@ -253,6 +253,14 @@ public static void ThrowIfIsNull(IntPtr ob)
             }
         }
 
+        internal static void ThrowIfIsNull(in NewReference reference)
```

Comment:
Just use `BorrowedReference` here (no need for `in` then). `NewReference` converts to it automatically (if not, you can add an implicit conversion).